src/main/java/uk/gov/justice/digital/delius/transformers/AppointmentPatchRequestTransformer.java

```diff
@@ -34,6 +35,15 @@ public static JsonPatch mapAttendanceFieldsToOutcomeOf(final ContextlessAppointm
         return new JsonPatch(patchOperations);
     }
 
+    public static JsonPatch mapOfficeLocation(final String officeLocation) {
+
+        final var patchOperations = new ArrayList<JsonPatchOperation>();
+
+        patchOperations.add(new ReplaceOperation(of(TARGET_OFFICE_LOCATION_FIELD_NAME), valueOf(officeLocation)));
+
+        return new JsonPatch(patchOperations);
+    }
```

Comment:
I'm guessing you could just write this as `return new JsonPatch(List.of(new ReplaceOperation(of(TARGET_OFFICE_LOCATION_FIELD_NAME), valueOf(officeLocation)))))`  ?